src/implementations/process/momenta.jl

```diff
@@ -12,13 +12,13 @@ Return four-momenta for incoming and outgoing particles for given coordinate bas
 function _generate_momenta(
```

Comment:
The function to be used to construct the momenta from coordinates is called ˋ_build_momenta ˋ for PSLs (see [here](https://github.com/AntonReinhard/QEDbase.jl/blob/481c85ed47f09542439b4d81f4a613757d316521/src/interfaces/phase_space_layout.jl#L154) for details).

This should also replace ˋgenerate_momenta ˋ everywhere, e.g. in the construction of PSPs from coordinates.

Reply:
I have deleted this function here since it serves no purpose anymore. The replacement of its calls will have to happen in QEDcore, it's never used in QEDbase so far. We should probably add a test for the PSP construction from coordinates so the interface can be tested, as you wrote below.

Reply:
You are right, of course, the construction of PSPs is specific for the concrete implementations. Therefore, this needs to be done in QEDcore.